composable_kernel/include/tensor_operation/blockwise_gemm_xdlops.hpp

```diff
@@ -233,7 +233,7 @@ template <index_t BlockSize,
           class BBlockDesc,
           index_t MPerWave,
           index_t NPerWave,
-          index_t KPack>
+          index_t K1>
```

Comment:
BlockwiseGemmXdlops_km_kn_m0m1m2n_v1_2x2pipeline need to be renamed to correctly reflect the dimensions